pkg/storage/system.go

```diff
@@ -28,7 +28,6 @@ const sqlListSystems = `
 		business_owner_component
 	FROM system_intakes
 	WHERE
-		status='LCID_ISSUED' AND
```

Comment:
double check

Reply:
removed a hard-coded conditional in SQL - is this ok?

Reply:
This function isn't in use, so I think you're OK here.

Reply:
I believe this is only used in [accessibility requests](https://github.com/CMSgov/easi-app/blob/main/src/views/Accessibility/AccessibilityRequest/Create/index.tsx#L20) in the FE, so it should be safe to modify. However if we wanted to replicate the query we would probably want `lcid_issued_at IS NOT NULL` or `decision_state = 'LCID_ISSUED'`.

Reply:
Lee's got it right here. This code is only used by the now-deprecated 508/Accessibility Request flow in EASi.


I don't think it's worth worrying about this outside just making it valid SQL, honestly.

After this review, I'm starting to think it's worth taking a swing at nuking some of the 508/HTTP&REST code, now that so much of it has become completely unmaintained.